css/queries.css

```diff
@@ -0,0 +1,154 @@
+/* Media queries */
+@media (min-width: 48em) {
```

Comment:
[OPTIONAL]
- [ ] What 48em means? Kindly consider using the `fixed(absolute) unit(px)` here. The `relative units`  are used when we need to style the children according to their parent size.